fvm/systemcontracts/system_contracts.go

```diff
@@ -0,0 +1,178 @@
+// Package systemcontracts stores canonical address locations for all system
```

Comment:
we keep other smart contracts and transactions that we need for protocol functionality inside the blueprints. wonder if we can include system ones there as well. thoughts?

Reply:
I'm a bit mixed on whether it makes sense to combine them. The new package has a very specific purpose which is just to keep track of which address we expect contracts to be deployed to, and doesn't care about what the contracts do or how to interact with them. But both packages are dealing with system contracts. I'm good either way.